spec/requests/api/v1/subscriptions_spec.rb

```diff
@@ -45,13 +47,13 @@
     end
   end
 
-  describe 'DELETE /subscriptions/terminate' do
+  describe 'delete /subscriptions/terminate' do
     let(:subscription) { create(:subscription, customer: customer, plan: plan) }
 
     before { subscription }
 
     it 'terminates a subscription' do
-      delete_with_token(organization, "/api/v1/subscriptions?customer_id=#{customer.customer_id}")
+      delete_with_token(organization, "/api/v1/subscriptions?subscription_id=#{subscription.id}")
```

Comment:
Maybe we could turn this route into `DELETE /api/v1/subscriptions/:id`. What do you think?

Reply:
Changed